packages/application-shell/src/configure-apollo.js

```diff
@@ -65,8 +43,35 @@ const fragmentMatcher = new IntrospectionFragmentMatcher({
   introspectionQueryResultData,
 });
 
-export const createApolloClient = () =>
-  new ApolloClient({
+export const createApolloClientForBackendApi = () => {
+  const httpLink = createHttpLink({
+    uri: `${window.app.mcApiUrl}/graphql`,
```

Comment:
This client connects to the MC API. The other one connects to the new graphql API in the frontend (proxy).

I couldn't come up with better names, so I went with `{backend,frontend}Client`